Modules/Challenges/test/Challenges.API.Tests.Performance.EndToEnd/src/utils.ts

```diff
@@ -0,0 +1,66 @@
+import http from "k6/http";
+import exec from "k6/execution";
+
+export interface Configuration {
+  Host: string;
+  Client_Secret: string;
+  User: string;
+  Password: string;
+  Size: string;
```

Comment:
Just noticed that the naming is incorrect here. The default in TypeScript is camelCase.